Markdown/Default.sublime-keymap

```diff
@@ -0,0 +1,41 @@
+[
+    // Auto-pair asterisks
+    { "keys": ["*"], "command": "insert_snippet", "args": {"contents": "*$0*"}, "context":
+        [
+            { "key": "setting.auto_match_enabled", "operator": "equal", "operand": true },
+            { "key": "selector", "operator": "equal", "operand": "text.html.markdown" },
```

Comment:
Markdown can include various code blocks and it wouldn't make sense to have those keybindings active in those (typically or always).

I'm not sure what would be the ideal selector but excluding `-source` would probably be a good start.

Reply:
Good point. I've excluded `markdown.raw`